recipes/fastq-multx/meta.yaml

```diff
@@ -1,13 +1,16 @@
+{% set version = "1.4.0" %}
+{% set github = " https://github.com/brwnj/fastq-multx" %}
```

Comment:
Jinja2 variables should only be used if they are actually used more than once in the recipe and for things that change regularly. Otherwise, their use just decreases readability.

Reply:
@marcelm Good call, removed the `github` variable and kept the `version` one.

Reply:
Thanks :-)